docker/kube/Dockerfile

```diff
@@ -15,14 +15,14 @@ RUN set -eux; \
 #}}} fpm
 
 #{{{ composer
-FROM composer:2 AS composer
+FROM fpm as composer
+COPY --from=composer:2.6.6 /usr/bin/composer /usr/local/bin/composer
```

Comment:
BTW. I don't know why we don't do that in all other Dockerfile.
This should speed up the build process if compared to those composer-installation steps we have in Deliver BE.